### ISSUE TITLE
New Eden Rebellion

### ISSUE THREAD

**Issue by Deathnetworks** [HerculesWS/Hercules#1173]
Appears to be an issue with the new eden NPC 'Blacksmith Thorn' in relation to Rebellion class characters... He doesn't seem to recognise rebellion class and cannot give quest gear.

Issue is located at - https://github.com/HerculesWS/Hercules/blob/1fd628e7be9e6de4ea48394be09ae10e912d8f3d/npc/re/quests/eden/eden_quests.txt#L1885

```
            if (Class == Job_Gunslinger) {
                callsub S_Select,"",
                    P_Revolver1,"Eden Revolver I","Revolver","HIT-5, attack 44","";
            }
```

Rebellion is not listed nor is it set to include baseclass in the statement

**Comment by AnisotropicDefixation** [comment]
Just change Class to BaseClass for Gungslinger checks. The job isn't supported by Hercules so such things aren't a priority.

**Comment by Ridley8819** [comment]
fixed in ae7a141
